src/Selenium2Library/keywords/_element.py

```diff
@@ -267,15 +267,31 @@ def element_text_should_be(self, locator, expected, message=''):
                           "in fact it was '%s'." % (locator, expected, actual)
             raise AssertionError(message)
 
-    def get_element_attribute(self, attribute_locator):
-        """Return value of element attribute.
+    def get_element_attribute(self, locator, attribute_name=None):
+        """Returns value of the element attribute.
 
-        `attribute_locator` consists of element locator followed by an @ sign
-        and attribute name, for example "element_id@class".
+        There are two cases how to use this keyword.
+
+        First, if only `locator` is provided, `locator` should consists of
+        element locator followed by an @ sign and attribute name.
+        This behavior is left for backward compatibility.
+
+        Example:
+        | ${id}= | Get Element Attribute | link=Link with id@id
+
+        Second, if `locator` and `attribute_name` are provided both, `locator`
```

Comment:
@aaltat I already know about PEP8 :) All the lines are not exceed 80 characters (length of the line above if 78 characters; except for the '+' sign, of course). But, anyway, because you requested to change the tables in the "example" sections I will check it again, just in case.